assets/oit_demo/shaders/FullscreenVS.hlsli

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+struct VSOutput
+{
+    float4 position : SV_POSITION;
+    float2 uv       : TEXCOORD;
+};
+
+VSOutput vsmain(uint VertexID : SV_VertexID)
+{
```

Comment:
Could `basic/shaders/FullScreenTriangle.hlsl` be reused? or is it different?

Reply:
The vertex shader is equivalent, but I use several different pixel shaders.
Also, I might actually have to add more interpolated values for other algorithms I plan to add.
Those interpolation would not make sense in the generic **FullScreenTriangle.hlsl**.
So, I'd rather keep my code for fullscreen passes separated for now, if that's ok.